src/nvim/normal.c

```diff
@@ -5028,6 +5029,14 @@ static void nv_ident(cmdarg_T *cap)
     g_tag_at_cursor = true;
     do_cmdline_cmd(buf);
     g_tag_at_cursor = false;
+
+    if (cmdchar == 'K' && !kp_ex && !kp_help) {
+      // Start insert mode in terminal buffer
+      restart_edit = 'i';
+
+      add_map((char_u *)"<buffer> <esc> <Cmd>call jobstop(&channel)<CR>", TERM_FOCUS, true);
+      do_cmdline_cmd("autocmd TermClose <buffer> bdelete!");
```

Comment:
even if it doesn't apply to pydoc, doesn't it make sense to skip this for non-zero `v:event.status` ?

Reply:
Yes, I have that change locally and was testing it when I made the discovery about pydoc. I'll push it soon.